src/packages/excalidraw/CHANGELOG.md

```diff
@@ -11,22 +11,24 @@ The change should be grouped under one of the below section and must contain PR
 Please add the latest change on the top under the correct section.
 -->
 
-## Unreleased
+## 0.7.0 (2021-04-26)
 
 ## Excalidraw API
 
 ### Features
 
-- Support scrolling to center to a single Excalidraw element, or not supplying any argument at all which will default to current elements on the scene [#3482](https://github.com/excalidraw/excalidraw/pull/3482).
+- Support scrolling to center to a single Excalidraw element, or not supplying any argument at all which will default to current elements on the scene using [`scrollToContent`](https://github.com/excalidraw/excalidraw/blob/master/src/components/App.tsx#L265) [#3482](https://github.com/excalidraw/excalidraw/pull/3482).
```

Comment:
```suggestion
- [`scrollToContent`](https://github.com/excalidraw/excalidraw/blob/master/src/components/App.tsx#L265) API now supports passing just a single Excalidraw element, or none at all (which will default to current elements on the scene) [#3482](https://github.com/excalidraw/excalidraw/pull/3482).
```

The previous change made it look like the `scrollToContent` API related to the "not supplying any argument" sentence.